astacus/coordinator/plugins/flink/flink.py

```diff
@@ -0,0 +1,42 @@
+"""Copyright (c) 2020 Aiven Ltd
```

Comment:
If this file is new, it is better to use 2022

Reply:
Fixed